exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Showroom.java

```diff
@@ -7,9 +7,9 @@ public static void main(String[] args) {
             Car car1 = new Car("Volvo","V40",2012);
             Car car2 = new Car("Porsche","Panamera",2009);
             Car car3 = new Car("Audi","A3",2018);
-            car1.getDetals();
-            car2.getDetals();
-            car3.getDetals();
+            System.out.print(car1.getDetals());
```

Comment:
Rather than appending a newline to the return value of `getDetals()` (with the implied assumption that it's going to be displayed on a command line), you could simply call `System.out.println()` which will append the desired newline for you.